frontend/src/main/main.js

```diff
@@ -92,21 +99,51 @@ class Info extends React.Component {
         super(props);
     }
 
+
+
     render() {
-        return (
-            <div className="col-3">
-                <p>Your mouse is {this.props.mouseover ? 'OVER' : 'NOT OVER'}  a bar on the viz!</p>
-                <p>The current viz color is {this.props.currentColor}</p>
-                <p>{this.props.person.length > 0 ? "The name of the person you clicked is: "
-                    + this.props.person + "\n": ''}</p>
-            </div>
-        );
+        if (this.props.showTableData) {
+            return (<div className="col-3">
+                <div className="row float-right">
+                    <button className="btn btn-primary" type="button" data-toggle="collapse"
+                        data-target="#toggleDisplayButton" id="toggle_button"
+                    >Toggle Display</button>
+                </div>
+                <div className="collapse row  float-right" id="toggleDisplayButton">
+                    <p>Your mouse is {this.props.mouseover ? 'OVER' : 'NOT OVER'}  a bar on the viz!</p>
+                    <p>The current viz color is {this.props.currentColor}</p>
+                    <table className="table">
+                        <tbody><tr>
+                            <th scope="row">Name:</th>
+                            <td>{this.props.person.length > 0 ? this.props.person : ""}</td>
+                        </tr>
+                        <tr>
+                            <th scope="row">Docs</th>
+                            <td>{this.props.docs > 0 ? this.props.docs : 0}</td>
+                        </tr>
+                        <tr>
+                            <th scope="row">Words</th>
+                            <td>{this.props.words > 0 ? this.props.words : 0}</td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>);
+        } else {
+            return (
+                <button id="toggle_button" onClick={()=>this.props.toggle_show_table()}>Toggle Display</button>
```

Comment:
readability nitpick -- arrow functions should have spaces like this:

`() => this.whatever()`